crates/typst/src/layout/layout.rs

```diff
@@ -27,11 +27,11 @@ use crate::syntax::Span;
 /// ])
 /// ```
 ///
-/// If the `layout` call is placed inside of a box width a width of `{800pt}`
-/// and a height of `{400pt}`, then the specified function will be given the
-/// parameter `{(width: 800pt, height: 400pt)}`. If it placed directly into the
-/// page it receives the page's dimensions minus its margins. This is mostly
-/// useful in combination with [measurement]($measure).
+/// If the `layout` call is placed inside a box width a width of `{800pt}` and a
```

Comment:
```suggestion
/// If the `layout` call is placed inside a box with a width of `{800pt}` and a
```
I know you didn't change it, but might as well fix it while you're at it :)

Reply:
Good catch!